src/apify/_actor.py

```diff
@@ -231,17 +231,19 @@ async def exit(
         event_listeners_timeout: timedelta | None = EVENT_LISTENERS_TIMEOUT,
         status_message: str | None = None,
         cleanup_timeout: timedelta = timedelta(seconds=30),
+        sys_exit: Callable = sys.exit,
```

Comment:
The type should specify the parameter and return value types as well
```suggestion
        sys_exit: Callable[[int], None] = sys.exit,
```